cabinets/__init__.py

```diff
@@ -41,7 +42,9 @@ def from_uri(uri) -> (Cabinet, str):
     try:
         protocol, path = uri.split('://')
     except ValueError:
-        raise InvalidURIError("Missing protocol identifier")
+        debug("No protocol identifier specified: using 'file'")
```

Comment:
This almost want to be a `warning`, but I think that implies something may have gone wrong, which may be ok, but it makes me feel like I did something wrong whereas we don't want to discourage this practice. I'd at least keep it at `info` since I think it's valuable to know even outside of debugging.
```suggestion
        info("No protocol identifier specified: using 'file'")
```